src/camvidlog2/plots.py

```
@@ -0,0 +1,19 @@
+
+import pandas as pd
+import seaborn as sns
+import seaborn.objects as so
+
+sns.set_theme(style="darkgrid")
+
+
+def plot_distances(distances: pd.DataFrame, outpath: str | Path) -> None:
+    # verify distances has structure and type expected
+    if "distance" not in distances.columns:
+        raise ValueError("Missing column 'distance'")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)